app/src/main/java/org/fossasia/openevent/app/data/models/Event.java

```diff
@@ -366,15 +369,85 @@ public void setVersion(Version version) {
     @Override
     public String toString() {
         return "Event{" +
-            "description='" + description + '\'' +
+            "backgroundImage='" + backgroundImage + '\'' +
+            ", description='" + description + '\'' +
             ", email='" + email + '\'' +
+            ", endTime='" + endTime + '\'' +
+            ", eventUrl='" + eventUrl + '\'' +
             ", id=" + id +
-            ", locationName='" + locationName + '\'' +
+            ", logo='" + logo + '\'' +
             ", name='" + name + '\'' +
-            ", organizerDescription='" + organizerDescription + '\'' +
             ", organizerName='" + organizerName + '\'' +
-            ", topic='" + topic + '\'' +
+            ", placeholderUrl='" + placeholderUrl + '\'' +
+            ", startTime='" + startTime + '\'' +
+            ", thumbnail='" + thumbnail + '\'' +
+            ", ticketUrl='" + ticketUrl + '\'' +
+            ", tickets=" + tickets +
+            ", timezone='" + timezone + '\'' +
+            ", type='" + type + '\'' +
             ", version=" + version +
             '}';
     }
+
+    // Parcelable Information - Only basic event info
+
+    @Override
+    public int describeContents() {
+        return 0;
+    }
+
+    @Override
+    public void writeToParcel(Parcel dest, int flags) {
```

Comment:
I checked the time to parcel - 1 ms

Changing parcel again and again will require extra overhead. I have only selected non nested string fields and tested for the speed